prow/crier/metrics.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package crier reports finished prowjob status to git providers.
+package crier
+
+import "github.com/prometheus/client_golang/prometheus"
+
+// Prometheus Metrics
+var (
+	// Use Fibonacci to get a good distribution. We care about report delayed
+	// for minutes, and even up to an hour.
+	reportTimeBuckets = []float64{1, 2, 3, 5, 8, 13, 21, 34, 55, 89, 144, 233, 377, 610, 987, 1597, 2584, 4181}
+
+	crierMetrics = struct {
+		reportTime *prometheus.HistogramVec
+	}{
+		reportTime: prometheus.NewHistogramVec(prometheus.HistogramOpts{
+			Name:    "report_time",
```

Comment:
Can we call this reporting delay or latency rather than time? I think that is an important distinction that we may care about later.

Reply:
renamed `latency`